resources/guides/laptop_guide.md

```diff
@@ -0,0 +1,72 @@
+
+-   8 GB of RAM
+    -   for gaming/future-proof purposes, 12 or 16 GB is more preferable
+-   256 GB of SSD storage
+    -   UWindsor provides 1 TB OneDrive cloud storage. However, if you download a lot of games and software, or if you simply prefer more local storage, you should upgrade to 512 GB / 1 TB storage
+
+## Operating System
+
+You aren't required to have certain OS since the School of Computer Science has remote Linux server that you can connect to, but here are a few pros and cons to help you decide which OS to use:
+
+### Windows
+
+Windows is one of the most popular OS among students, particularly because it comes with most pre-assembled laptops. The user interface should be fairly intuitive even when you are new to using laptops, and it is one of the most supported OS for app availability or IT support. Additionally, it is much easier to game on Windows.
+
+Many Computer Science classes (COMP-1400 and COMP-1410, for example) will require programming environments that are harder to set up in Windows than in other UNIX/Linux-based OS. Although substitutes like WSL and virtual machines can help, they may not be fully compatible or require a lot of computing power. Should you choose Windows as your primary OS, you should consider setting up an SSH connection to the UWindsor's remote server on editors like Visual Studio Code or via other means. In addition, when you are working with other developers on a project, you may also have to pay attention to your line breaks (CRLF vs. LF).
+
+If you are taking classes offered by other faculties or departments (e.g. Engineering), you may have to install software that are only available on Windows. However, this usually is not a concern since such instances are quite rare, and the University of Windsor has many other computer resources to help you out if something like this happens.
+
+### Linux
+
+Linux/UNIX is very good for programming/developing because of how easy it is to set up the necessary environment. Much like MacOS, it can be difficult to game on. Depending on how important gaming is to you, you may need to use Wine, set up virtual machine, or dual-boot with Windows. Additionally, since many of the University's software (Office 365, OneDrive, Teams, etc.) are on the Microsoft platform, there _may_ be some compatibility issues.
+
+Popular Linux distros include Ubuntu, Debian, Fedora, Pop!\_OS, Linux Mint, and Arch Linux. If you are very new to Linux, then Ubuntu and Debian may be a good starting point for you simply because there is a large community using them and most likely have answers for any problem you encounter. The University, for example, uses Debian on its remote servers. If you have time and are willing to experiment, you should test out the different Linux OS since the desktop environment on Ubuntu and Debian (GNOME) can be slightly laggy on laptops without high specifications.
```

Comment:
```suggestion
Popular Linux distros include Ubuntu, Debian, Fedora, Pop!\_OS, Linux Mint, and Arch Linux. If you are very new to Linux, then Ubuntu and Debian may be a good starting point for you simply because there is a large community using them and most likely have answers for any problem you encounter. The University, for example, uses Debian on its remote servers. If you have time and are willing to experiment, you should test out the different Linux OS since the desktop environment on Ubuntu and Debian (GNOME) can be slightly laggy on laptops without high specifications. If you are coming from Windows, Linux Mint's Cinnamon desktop environment will help you adjust to Linux as it has a look and feel comparable to that of Windows.
```